ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -101,6 +102,13 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
 		volumeWithHost: make(map[string]string), // map with key:=hostSourcePath value:=VolumeName
 	}
 
+	// Add named volume configs:
+	if context.Project.VolumeConfigs != nil {
```

Comment:
This is fine and it works, but it would follow the style of the other code (and possibly make future changes easier, including support for Compose V3 syntax) if the VolumeConfigs were added as an argument to `ConvertToTaskDefinition` and a `VolumeConfigs()` getter function was added to `project.go`.

See: https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/cli/compose/project/project.go#L166